.env.example

```
@@ -0,0 +1,3 @@
+MONGO_URI=mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASS}@cluster0.y15rh.mongodb.net/diucseapi?retryWrites=true&w=majority
+JWT_SECRET=diuCSEalumniSecret
+PORT=3000
```

```suggestion
MONGO_URI=
JWT_SECRET=
PORT=
```

Keep the .env.example file empty. Just expose the variable so that .env file can be populated